easybuild/easyblocks/s/scotch.py

```diff
@@ -108,7 +117,7 @@ def build_step(self):
         if self.toolchain.options['i8']:
             cflags += " -DINTSIZE64"
 
-        if not self.toolchain.mpi_family() in [toolchain.INTELMPI, toolchain.QLOGICMPI]:  #@UndefinedVariable
+        if self.cfg['threadedmpi']: 
```

Comment:
@ocaisa this implies that for builds on top of OpenMPI (for example), `threadedmpi = True` must be now specified to obtain the same installation, so this should be something like:

```python
if self.cfg['threadedmpi'] or if self.toolchain.mpi_family() not in [toolchain.INTELMPI, toolchain.QLOGICMPI]
```

to ensure backward compatibility?